python/ql/lib/change-notes/2023-01-16-new-call-graph.md

```diff
@@ -0,0 +1,4 @@
+---
+category: majorAnalysis
+---
+* We use a new analysis for the call-graph (determining which function is called). This can lead to changed results. In most cases this is much more accurate than the old call-graph that was based on points-to, but we do loose a few valid edges in the call-graph, especially around methods that are not defined inside its' class.
```

Comment:
I guess we also lose the few special methods we had modelled, but I am not sure it is worth mentioning here.

Reply:
True. I think I'll leave it as is then if you don't feel more strongly about it, since it's broad enough to cover the special methods as well.

Reply:
Agreed